File: sol/solution/0001-0100/0038. Count and Say/Solution.py
class Solution:
    def countAndSay(self, n: int) -> str:
        res = "1"
        for _ in range(1, n):
            cur = ""
            cnt = 1
            for j in range(1, len(res)):
                if res[j] == res[j-1]:
                    cnt += 1
                else:
                    cur += str(cnt) + res[j-1]
                    cnt = 1
            cur += str(cnt) + res[-1]
            res = cur
        return res

File: sol/solution/0001-0100/0038. Count and Say/Solution.php
class Solution {
    /**
     * @param Integer $n
     * @return String
     */
    function countAndSay($n) {
        $res = "1";
        for ($i = 2; $i <= $n; $i++) {
            $cur = "";
            $cnt = 1;
            for ($j = 1; $j < strlen($res); $j++) {
                if ($res[$j] === $res[$j-1]) {
                    $cnt++;
                } else {
                    $cur .= $cnt . $res[$j-1];
                    $cnt = 1;
                }
            }
            $cur .= $cnt . $res[strlen($res)-1];
            $res = $cur;
        }
        return $res;
    }
}